Source/WebCore/rendering/svg/RenderSVGResourceClipper.cpp

```diff
@@ -191,6 +196,23 @@ void RenderSVGResourceClipper::applyMaskClipping(PaintInfo& paintInfo, const Ren
     frameView.setPaintBehavior(oldBehavior);
 }
 
+void RenderSVGResourceClipper::calculateClipContentRepaintRect(RepaintRectCalculation repaintRectCalculation)
```

Comment:
That should be moved into SVGClipPathElement, just like shouldApplyPathClipping, no?

Reply:
Sorry, missed this one, will fix.